Task7FinalReact/valery-youtube-searcher/src/actions/videos.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios'
+import {addVideos, setFetchError, setIsFetching, setNextPageToken, setVideos} from "../reducers/videosReducer";
```

Comment:
Please, use absolute paths everywhere. It means it should be `src/reducers/videosReducer` or even `reducers/videosReducer` (it depends on how you configure it).

You should adjust webpack config for achieving it.